packages/typegpu/src/data/std140.ts

```diff
@@ -23,6 +23,7 @@ export class SimpleTgpuData<TSchema extends AnySchema>
   public readonly size: number;
   public readonly byteAlignment: number;
   public readonly expressionCode: string;
+  isCustomAligned = false;
```

Comment:
Nitpick, for consistency
```suggestion
  public readonly isCustomAligned = false;
```